frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -907,8 +907,6 @@ export const EventManager = ({
     );
   };
 
-  const componentName = componentMeta.name ? componentMeta.name : 'query';
-
   if (events.length === 0) {
```

Comment:
Hi @Avi98 the ask here is to remove the text only in the column properties widget. This is a common component which is reused across app. So this change will cause that text to disappear everywhere rather than only in aforementioned widget. You can introduce a new prop to this component (eg: like `hideEmptyEventAlert`) and if its value is true/false hide the warning text. Then from the config popup of table u can pass it as true and make it hidden only for the specific widget.

Reply:
Thanks @johnsoncherian, done